### PR TITLE
CORTX-33540 : Added the cron entry for odd day nightly run at 22:00 and setup email notification

### PR DESCRIPTION
**Pull request by Abhirup-A-G**
Signed-off-by: Abhirup Ghosh <abhirup.ghosh@seagate.com>

# Problem Statement
- Make the job executable for odd days during night time
- Set up post job completion notifications 

# Design
-  For Feature: setup the cron entry the Job to run on odd day at 22:00 and configured the email notification.

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [x] Unit and System Tests are added
- [x] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [x] Testing was performed with RPM
- [x] Jenkins pipelines used for testing  : https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/AbhirupSpace/job/community_build_job2_/54/
image

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [x] Interface change (if any) are documented
- [x] Side effects on other features (deployment/upgrade)
- [x] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by pankajgborole — CHANGES_REQUESTED** [review]
Please schedule to run this only Mon/Web/Friday

**Review by gauravchaudhari02 — APPROVED** [review]

**Review by pankajgborole — APPROVED** [review]
Approved

File: jenkins/deployment/community/cortx-community-deploy.groovy [hunk 1]
```diff
@@ -4,7 +4,7 @@ pipeline {
             label 'community-build-executor'
         }
     }
-    
+    triggers { cron('0 22 * * 1,3,5') }
```
Comment (shailesh-vaidya) [inline jenkins/deployment/community/cortx-community-deploy.groovy:7]: Just to confirm this is not a daily cron schedule. Please check

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]

File: jenkins/deployment/community/cortx-community-deploy.groovy (hunk 1 above)
Comment (Abhirup-A-G) [reply jenkins/deployment/community/cortx-community-deploy.groovy:7]: @shailesh-vaidya , I have updated the title and description of the PR. Pankaj requested for > https://github.com/Seagate/cortx-re/pull/1131#pullrequestreview-1041760530

**Review by shailesh-vaidya — APPROVED** [review]

**Pull request merged**